tests/helpers/test_entity_registry.py

```diff
@@ -1605,9 +1607,10 @@ def test_migrate_entity_to_new_platform(
 
 
 async def test_restore_entity(
-    hass: HomeAssistant, entity_registry: er.EntityRegistry, update_events, freezer
+    hass: HomeAssistant, entity_registry: er.EntityRegistry, freezer
```

Comment:
```suggestion
    hass: HomeAssistant, entity_registry: er.EntityRegistry, freezer: FrozenDateTimeFactory
```
Might as well type everything

Reply:
Nice catch!